components/microsoft_outlook/microsoft_outlook.app.mjs

```diff
@@ -14,49 +18,280 @@ export default {
       description: "Subject of the email",
       type: "string",
     },
+    contentType: {
+      label: "Content Type",
+      description: "Content type (default `HTML`)",
+      type: "string",
+      optional: true,
+    },
     content: {
       label: "Content",
       description: "Content of the email in text format",
       type: "string",
       optional: true,
     },
-    name: {
-      label: "File name",
-      description: "Name of the file seen by the recipients (eg. `new_file_name.pdf`)",
+    files: {
+      label: "File paths",
+      description: "Absolute paths to the files (eg. `/tmp/my_file.pdf`)",
+      type: "string[]",
+      optional: true,
+    },
+    timeZone: {
+      label: "Time Zone",
+      description: "Time zone of the event in supported time zones, [See the docs](https://docs.microsoft.com/en-us/graph/api/outlookuser-supportedtimezones)",
+      type: "string",
+      async options() {
+        const timeZonesResponse = await this.getSupportedTimeZones();
+        return timeZonesResponse.value.map((tz) => ({
+          label: tz.displayName,
+          value: tz.alias,
+        }));
+      },
+    },
+    contact: {
+      label: "Contact",
+      description: "The contact to be updated",
+      type: "string",
+      async options() {
+        const contactResponse = await this.listContacts();
```

Comment:
Shouldn't this one support pagination?

Reply:
Hi @jcortes tl;tr; it is not supported.
I double checked pagination [docs](https://docs.microsoft.com/en-us/graph/paging). It says that some of the resources does not support pagination. When I check [contacts](https://docs.microsoft.com/en-us/graph/api/user-list-contacts) it seems it is not supported and when I check [users](https://docs.microsoft.com/en-us/graph/api/user-list) as another example, it seems supported there. If you find more information please let me know. I left it as it was for now.